### PR TITLE
Kubernetes v1.22 Code Freeze

### PR DESCRIPTION
**Pull request by thejoycekung**
This PR enables code freeze for `kubernetes/kubernetes` on the master branch.

/hold
This PR should only be merged after Thursday, July 8, 2021 18:00 PDT.

/sig release
/area release-eng

cc: @kubernetes/release-team-leads @kubernetes/release-engineering

### CODE REVIEW

File: config/prow/config.yaml [hunk 1]
```diff
@@ -535,6 +535,29 @@ tide:
     - needs-rebase
   - repos:
     - kubernetes/kubernetes
+    excludedBranches:
+    - master
```
Comment (puerco) [inline config/prow/config.yaml:539]:
```suggestion
    - master
    - release-1.22
```
I just noticed a little error here, we have to add the 1.22 branch (`release-1.22`) to this exclusion and also to the requirements below. The branch does not exist yet, but we want code freeze to also apply to it when we create it in `v1.22.0-rc.0`.

File: config/prow/config.yaml [hunk 2]
```diff
@@ -535,6 +535,29 @@ tide:
...
+    - do-not-merge/needs-kind
+    - do-not-merge/needs-sig
+    - do-not-merge/release-note-label-needed
+    - do-not-merge/work-in-progress
+    - needs-rebase
+  - repos:
+    - kubernetes/kubernetes
+    milestone: v1.22
+    includedBranches:
+    - master
```
Comment (puerco) [inline config/prow/config.yaml:560]: Same thing here:
```suggestion
    - master
    - release-1.22
```

**Review by puerco — CHANGES_REQUESTED** [review]
Thanks @thejoycekung , just one nit, we also need to add the 1.22 branch to avoid making a second PR when we create the branch :point_down:

**Review by cpanato — APPROVED** [review]
/lgtm

**Review by xmudrii — APPROVED** [review]

**Comment by puerco** [comment]
:snowflake:  **Code Freeze is now in effect.**
/cc @kubernetes/release-team 
Thank you @thejoycekung!

**Pull request merged**